CONTRIBUTING.md

```diff
@@ -33,3 +33,39 @@ accept your pull requests.
    recommended coding standards for this organization.
 1. Ensure that your code has an appropriate set of unit tests which all pass.
 1. Submit a pull request.
+
+
+## Best Practices
+
+The following best-practice guidelines will help ensure your initialization
+actions are less likely to break from one Dataproc version to another, and
+most likely to support different single-node, high-availability, and
+standard cluster modes.
+
+1. Where possible, install from Dataproc's prebuilt Debian packages (built
+   using Apache Bigtop) instead of installing from tarballs. Dataproc packages
+   can be found under /var/lib/apt/lists/*dataproc-bigtop-repo_*Packages on
```

Comment:
Should they install them from here or should they rely on `apt-get install <package>`?

Reply:
Added clarification. Indeed they should just use "apt-get install", I just wanted to explain that apt-get install grabs from Dataproc's distro (resolving conflicts between some other Hadoop-related debian package vs Dataproc's sources could be a more advanced matter to explain).